code/unit_tests/spawner_tests.dm

```diff
@@ -0,0 +1,38 @@
+/*
+ *
+ *  Unit Test Template
+ *  This file is not used.
+ *
+ */
+
+datum/unit_test/template
+	name = "Ghost Spawner Tests"		// If it's a template leave the word "template" in it's name so it's not ran.
+	
+
+datum/unit_test/template/start_test()
+	var/list/ignore_spawners = list(
+		/datum/ghostspawner/human,
+		/datum/ghostspawner/human/admin,
+		/datum/ghostspawner/simplemob
+		)
+	var/failed_checks = 0
+	var/checks = 0
+
+	for(var/spawner in subtypesof(/datum/ghostspawner))
+		checks++
+		var/datum/ghostspawner/G = new spawner
+		if(is_type_in_list(G,ignore_spawners))
```

Comment:
this doesn't work. It executes `istype`, which will mean that all subtypes of the ignored_spawners list will get ignored. As opposed to the explicit types noted.

Either use a typecache, or refactor this to the following:
```suggestion
		if(instances_of_type_in_list(G,ignore_spawners, strict = TRUE))
```

Reply:
You might also want to print out the number of spawners actually checked. And verify it.